### DISCUSSION TITLE
All video displaying as white, yellow, and blue, extremely bright

### DISCUSSION THREAD

**Discussion by neocrystal** [smplayer-dev/smplayer#1036] [Q&A] [answered]
Hello, recently have been having a problem with smplayer while in mpv mode. The problem doesn't happen on base mpv nor does it happen on smplayer in mplayer mode, and I'm wondering how I can fix this problem? Attached is a file exampling what I'm talking about, thank you.
![cap_unknown_replay_2024 06 23-16 24_1_00_00_00_01](https://github.com/user-attachments/assets/13899d9a-aed7-4b7d-92af-742f2505eb7a)

**Comment by Raza-0007** [comment, marked as answer]
Which video output driver are you using? Switch to gpu-next and see if it corrects your issue.

~Raza

**Reply by neocrystal** [reply]
i was using default, just switched to gpu-next and it didnt solve the issue, direct3d seems to work though, thank you very much!
